qemu/tests/seabios_reboot_timeout.py

```diff
@@ -0,0 +1,51 @@
+import re
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    KVM seabios test:
+    1) Start VM
+    2) Check if reboot-timeout option works as expected
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_output(session_obj):
+        """
+        Use the function to short the lines in the scripts
+        """
+        return session_obj.get_stripped_output()
+
+    def reboot_timeout_check():
+        """
+        reboot-timeout check
+        """
+        return re.search(pattern, get_output(seabios_session), re.S)
```

Comment:
@vivianQizhu  have you look through the script?
The implementation of `get_output`  and `reboot_timeout_check` does not look promising.

Reply:
@hereischen Sorry, I don't get you, I see it trying to find the pattern as polarion test case described, which looks okay to me. Would you help explain more? Thanks.